firmware/config/boards/common_make.sh

```diff
@@ -18,7 +18,8 @@ FW_DIR=$(readlink -f $FW_DIR)
 echo "FW dir is $FW_DIR"
 cd $FW_DIR
 
-mkdir -p .dep
+mkdir -p .dep # ChibiOS build's DEPDIR
+mkdir -p build # ChibiOS build's BUILDDIR
```

Comment:
this is actually configurable via `BUILDDIR`, as is I think `DEPDIR`, but these features are hardcoded in a few places, mainly around the `compile.sh` layer atop Make